### PR TITLE
CD-411 Typography deploy to Feature

### PR DESCRIPTION
**Pull request by left23**
Do not merge. This is to deploy the typography changes to the Feature site for testing.

chore: use branch of base theme to deploy for testing

Refs: CD-411

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,6 +21,10 @@
         {
             "type": "composer",
             "url": "https://asset-packagist.org"
+        },
+        {
+            "type": "vcs",
+            "url": "git@github.com:UN-OCHA/common_design.git"
```
Comment (cafuego) [inline composer.json:27]:
```suggestion
            "url": "https://github.com/UN-OCHA/common_design.git"
```

**Review by cafuego — APPROVED** [review]
I approve now.

File: .travis.yml [hunk 2]
```diff
@@ -31,12 +31,12 @@ before_script:
   # Ensure the PHP environment is ready.
   - phpenv rehash
 
+  # You *will* use HTTPS.
```
[the remark this answers is not in the document]
Comment (cafuego) [reply .travis.yml:34]: It shouldn't break anything, so it's safe to merge I think.

**Comment by left23** [comment]
Merging this, see comments above.

**Pull request merged**